region-capture.js

```diff
@@ -1,6 +1,6 @@
 var respecConfig = {
   group: "webrtc",
-  specStatus: "ED",
+  specStatus: "FPWD",
```

Comment:
```suggestion
  specStatus: "ED",
```
The version in the repo should remain as "ED" - it is when it gets published under /TR/mediapture-region that the proper status will be set through the publication system.